driver/configurations/cache.cmake

```
@@ -392,9 +392,11 @@ else()
       OUTPUT_VARIABLE DASHBOARD_LSB_RELEASE_DESCRIPTION_OUTPUT_VARIABLE
       OUTPUT_STRIP_TRAILING_WHITESPACE
     )
+    # NOTE: Focal reports `Ubuntu 20.04.4 LTS` whereas Jammy reports
+    # `Ubuntu 22.04 LTS`, the third number must be optional.
     if(DASHBOARD_LSB_RELEASE_DESCRIPTION_RESULT_VARIABLE EQUAL 0 AND
        DASHBOARD_LSB_RELEASE_DESCRIPTION_OUTPUT_VARIABLE MATCHES
-       "^Ubuntu ([0-9]+[.][0-9]+[.][0-9]+) LTS$"
+       "^Ubuntu ([0-9]+[.][0-9]+([.][0-9]+)*) LTS$"
```

If this is optional, a more correct regex would be:
```suggestion
       "^Ubuntu ([0-9]+[.][0-9]+([.][0-9]+)?) LTS$"
```
Or you could just generalize it to support any number of... numbers:
```suggestion
       "^Ubuntu ([0-9]+([.][0-9]+)+) LTS$"
```

---

Went with the second one :+1: